src/v/pandaproxy/schema_registry/store.h

```diff
@@ -84,6 +84,23 @@ class store {
           .deleted = v_it->deleted};
     }
 
+    ///\brief Return a list of versions and associated schema_id.
+    result<std::vector<schema_version>> get_versions(const subject& sub) {
+        auto sub_it = _subjects.find(sub);
+        if (sub_it == _subjects.end()) {
+            return error_code::subject_not_found;
+        }
+        const auto& versions{sub_it->second};
```

Comment:
is this a syntax that is preferred sometimes over `=`?

Reply:
Not in this case. Changed.